frontend/src/components/pages/topics/DeleteRecordsModal/DeleteRecordsModal.tsx

```diff
@@ -277,7 +278,7 @@ const ManualOffsetContent = observer(
                     className={styles.slider}
                 />
                 <Input
-                    className={styles.sliderValue}
+                    maxWidth={124}
```

Comment:
Why specifically this value? Just curious, does it look better?

Reply:
https://github.com/redpanda-data/console/pull/909/files#diff-d349e3386d50bc0993600dd1ae2a8530c0ec6bd7005f2ade392a1257d25f3e76L18 I've migrated 1:1. We need however introduce some consistency globally using a styleguide.